src/components/GenerativeDirectAnswerDisplay.tsx

```diff
@@ -0,0 +1,137 @@
+import { 
+  GenerativeDirectAnswerResponse, 
+  useSearchActions, 
+  useSearchState, 
+  SearchTypeEnum, 
+  Result 
+} from '@yext/search-headless-react';
+import { useComposedCssClasses } from '../hooks';
+import { executeGenerativeDirectAnswer } from '../utils/search-operations';
+import ReactMarkdown from "react-markdown";
+import React from 'react';
+
+/**
+ * The CSS class interface used for {@link GenerativeDirectAnswerDisplay}.
+ *
+ * @public
+ */
+export interface GenerativeDirectAnswerDisplayCssClasses {
+  generativeDirectAnswerTitle?: string,
+  generativeDirectAnswerText: string,
+  generativeDirectAnswerDivider: string,
+  generativeDirectAnswerContainer?: string,
+  generativeDirectAnswerCitations: string,
+  generativeDirectAnswerCitation?: string,
```

Comment:
It may not be necessary to include `generativeDirectAnswer` prefix since the component is already named that, can you check the patterns from other existing components to see the naming format? I think only some have the component name prefix